GTBitsOfGood/national-npp
# 50 view project applications

## What is this?
This pull request includes:
- User interface changes for the chapter project table to match Figma designs
- Loading chapter projects that have the "Application Review" status
- Error notifications where needed


## Setup



## Steps to Test


- In MongoDB, make sure your user profile has an associated chapter attached to it
- In MongoDB there are projects associated with the same chapter
- Navigate to http://localhost:3000/chapter/applications
- Check if projects in the "Application Review" status are viewable

## Associated Issues

- #50 



## Additional Instructions

- Nonprofit name + email appears to be undefined (even though the database structure looks okay) and therefore does not appear in the table.
